### PR TITLE
Added buttons and door

### PR DESCRIPTION
**Pull request by ianxcloud**

### CODE REVIEW

File: VHS Hero/Assets/Scripts/Misc/Restart.cs [hunk 1]
```diff
@@ -1,14 +0,0 @@
...
-using System.Collections.Generic;
-using UnityEngine;
-
-public class Restart : MonoBehaviour
-{
-    private void Update()
-    {
-        if (Input.GetKeyDown(KeyCode.R))
-        {
-            GameObject.Find("levelman").GetComponent<LevelManager>().Restart();
```
Comment (maxall41) [inline VHS Hero/Assets/Scripts/Misc/Restart.cs:11]: Why was this file removed?

**Review by maxall41 — APPROVED** [review]

**Comment by ianxcloud** [comment]
I put the method in
VHS-Hero-git<https://github.com/ianx64/VHS-Hero-git>/VHS Hero<https://github.com/ianx64/VHS-Hero-git/tree/main/VHS%20Hero>/Assets<https://github.com/ianx64/VHS-Hero-git/tree/main/VHS%20Hero/Assets>/scripts<https://github.com/ianx64/VHS-Hero-git/tree/main/VHS%20Hero/Assets/scripts>/Gameplay<https://github.com/ianx64/VHS-Hero-git/tree/main/VHS%20Hero/Assets/scripts/Gameplay>/Player<https://github.com/ianx64/VHS-Hero-git/tree/main/VHS%20Hero/Assets/scripts/Gameplay/Player>/TimeControl.cs
with other input events for easier management.


On 7 Jul 2021, at 4:22 am, maxall41 ***@***.***> wrote:

﻿

@maxall41 commented on this pull request.

________________________________

In VHS Hero/Assets/Scripts/Misc/Restart.cs<https://github.com/maxall41/VHS-Hero-git/pull/5#discussion_r664782683>:

> @@ -1,14 +0,0 @@
-using System.Collections;
-using System.Collections.Generic;
-using UnityEngine;
-
-public class Restart : MonoBehaviour
-{
-    private void Update()
-    {
-        if (Input.GetKeyDown(KeyCode.R))
-        {
-            GameObject.Find("levelman").GetComponent<LevelManager>().Restart();


Why was this file removed?
